hack/app_sre_build_deploy.sh

```diff
@@ -7,19 +7,20 @@ set -exv
 CURRENT_DIR=$(dirname "$0")
 
 BASE_IMG="aws-account-shredder"
-QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
-IMG="${BASE_IMG}:latest"
+QUAY_IMAGE="quay.io/alima/${BASE_IMG}"
```

Comment:
This should be reverted.

Reply:
All changes to hack need to be reverted, thank you for pointing it out